openedx/features/edly/utils.py

```diff
@@ -333,3 +335,31 @@ def create_learner_link_with_permission_groups(user):
         user.groups.add(new_group)
 
     return user
+
+
+def get_current_site_invalid_certificate_context(default_html_certificate_configuration):
+    """
+    Gets current site's context data for invalid certificate.
+
+    Try to get current site's context data for invalid certificate from site configuration
+    or fallback to empty urls.
+
+    Arguments:
+        default_html_certificate_configuration (dict): Default html configurations dict.
+
+    Returns:
+        dict: Context data.
+    """
+    context = dict(default_html_certificate_configuration.get('default'))
+    current_site_configuration = get_current_site_configuration()
+
+    if not current_site_configuration:
+        return context
+
+    context['platform_name'] = current_site_configuration.get_value('platform_name', settings.PLATFORM_NAME)
+    context['logo_src'] = get_logo_url()
+    logo_redirect_url = settings.LMS_ROOT_URL
+    context['logo_url'] = logo_redirect_url
```

Comment:
Instead of storing the value in variable we can do this `context['logo_url'] = settings.LMS_ROOT_URL`

Reply:
`logo_url` sounds like url for logo, the intention to use `logo_redirect_url` is to show clearly this is logo redirect URL not logo src url.